deploy/aws-lambda/README.md

```diff
@@ -0,0 +1,9 @@
+## Deploy in a AWS Lambda Function
+
+You can deploy the webhook to an AWS Lambda function and activate it in your Kubernetes cluster by adding the following module to your Terraform file: 
+
+```terraform
+module "secrethub_mutating_webhook" {
+  source = "github.com/secrethub/secrethub-kubernetes-mutating-webhook/deploy/aws-lambda"
+}
+```
```

Comment:
We should mention that it depends on the AWS and Kubernetes providers:

````suggestion
```

This module requires the [AWS provider](https://registry.terraform.io/providers/hashicorp/aws/latest) and [Kubernetes provider](https://registry.terraform.io/providers/hashicorp/kubernetes/latest) to be configured in your root project.
````